### PR TITLE
Update to use Node 14.x

### PR DESCRIPTION
**Pull request by maestromac**
## What type of PR is this? (check all applicable)
- [x] DX

## Description
Prefer a more widen node version. Originally implemented by @citizen428 

## Related Tickets & Documents
Extracted from https://github.com/forem/forem/pull/12103

## QA Instructions, Screenshots, Recordings
n/a

## Added tests?
- [x] no, because they aren't needed

## Added to documentation?
- [x] no documentation needed

## [optional] Are there any post deployment tasks we need to perform?
n/a

### CODE REVIEW

File: Containerfile [hunk 1]
```diff
@@ -2,7 +2,8 @@ FROM quay.io/forem/ruby:2.7.2 as builder
 
 USER root
 
-RUN curl -sL https://dl.yarnpkg.com/rpm/yarn.repo -o /etc/yum.repos.d/yarn.repo && \
+RUN curl -sL https://rpm.nodesource.com/setup_lts.x && \
```
Comment (jdoss) [inline Containerfile:5]: Why are we adding this repo in when the main Fedora repo provides Node?

**Review by jdoss — CHANGES_REQUESTED** [review]

File: Containerfile (hunk 1 above)
Comment (maestromac) [reply Containerfile:5]: Ah, I extracted this code from a different PR and I wasn't aware that it's redundant. Removing!

**Review by nickytonline — APPROVED** [review]
LGTM. 🚢

File: Containerfile (hunk 1 above)
Comment (citizen428) [reply Containerfile]: @jdoss Does Fedora only include LTS versions? If yes, this is indeed not needed.

Comment (jdoss) [reply Containerfile]: IIRC they track LTS. Regardless, we should just follow whatever node version that is currently stable in Fedora as a forcing function to upgrade.

**Review by jdoss — APPROVED** [review]

**Pull request merged**